tests/main/test_grid_init.py

```diff
@@ -26,7 +26,13 @@ def get_quantity_factory(layout, nx_tile, ny_tile, nz):
     nx = nx_tile // layout[0]
     ny = ny_tile // layout[1]
     return QuantityFactory.from_backend(
-        sizer=SubtileGridSizer(nx=nx, ny=ny, nz=nz, n_halo=3, extra_dim_lengths={}),
+        sizer=SubtileGridSizer(
+            nx_tile=nx,
+            ny_tile=ny,
```

Comment:
This is the constructor, not `from_tile_params()`. 

```suggestion
            nx=nx,
            ny=ny,
```

Reply:
nevermind - I was too late. Should be all good now.